os/kernel/log_dump/log_dump.c

```diff
@@ -47,11 +51,14 @@
 #endif
 
 #define LOG_DUMP_OK			 0
+#define LOG_DUMP_OK_FSZ			 1
 #define LOG_DUMP_MEM_FAIL		-1
 #define LOG_DUMP_HEAP_FAIL		-2
 #define LOG_DUMP_SIZE_FAIL		-3
+#define LOG_DUMP_STR_FREEMEM		15
```

Comment:
I can't understand what they mean. FSZ and STR_FREEMEM.

Reply:
I have renamed these, FSZ is free heap size checking and STR_FREEMEM was for string used to print the free heap size along with logs